test_pull_requests

```diff
@@ -412,8 +412,8 @@ module Hub
 
       repo_to_pull_request.each do |repo, pull_request|
         if repo != base_repo
-          process_or_create_comment(pull_request['number'], repo, settings) do |comment_id, comment, comment_updated_at|
-            update_comment(comment_id, repo, running_comment)
+          process_or_create_comment(pull_request['number'], repo, settings) do |bot_comment_id, comment, comment_updated_at|
```

Comment:
My objection is you are using the verbose name that doesn't separate it was the name above.  The less verbose name for the inner context is a common pattern.